packages/components/src/components/linear-progress/linear-progress-container.jsx

```diff
@@ -2,61 +2,46 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { LinearProgress } from './linear-progress.jsx';
 
-class LinearProgressContainer extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            timeout: props.timeout / 1000,
-            total_ticks: Math.round(props.timeout / 1000),
-            current_tick: Math.round(props.timeout / 1000),
-        };
-    }
+const LinearProgressContainer = ({ timeout, action, render, className }) => {
+    const [timeout_state, setTimeout] = React.useState(timeout);
```

Comment:
Why is the initial state different before the refactor?
In the class component version it's `timeout / 1000`.

Reply:
True, I mistakenly changed it. Fixed it now.